MyAddin/Form1.cs

```diff
@@ -0,0 +1,317 @@
+﻿using System;
+using System.IO;
+using System.Windows.Forms;
+using System.Xml;
+
+namespace MyAddin
+{
+    public partial class Form1 : Form
+    {
+        private XmlDocument docXML = new XmlDocument();
+
+        //this is the path where xml files are saved and loaded
+        //the input and output file will be found in AppData/roaming folder
+        string path = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+        string fileName = Path.Combine(Environment.GetFolderPath(
+                Environment.SpecialFolder.ApplicationData), "input.xml");
+
+        string fileName1 = Path.Combine(Environment.GetFolderPath(
+                Environment.SpecialFolder.ApplicationData), "GenericFTS.xml");
+        String name1 = "";
```

Comment:
In C#, `string` is an alias for `String`. Change it to `string` to remain consistent with rest of code.